examples/multi-platform-data/projects/databricks-delta/src/databricks_delta/definitions.py

```diff
@@ -0,0 +1,5 @@
+import dagster as dg
+
+import databricks_delta.defs as defs_module
+
+defs = dg.components.load_defs(defs_module)
```

Comment:
is there a reason to prefer this over `load_defs_from_folder()`? I feel like standardizing on that is ideal